import/webview/plugin.cpp

```diff
@@ -74,6 +83,68 @@ void SailfishOSWebViewPlugin::initializeEngine(QQmlEngine *engine, const char *u
     webEngine->addObserver(QStringLiteral("clipboard:setdata"));
 }
 
+void SailfishOSWebViewPlugin::initUserAgentOverrides(const QString &path)
```

Comment:
This could be a static function (it doesn't access any class members from what I can tell), but it's a small point; your choice whether to change or not.